ios/Sources/FavoritesFeature/FavoritesScreen.swift

```diff
@@ -1,53 +1,67 @@
 import ComposableArchitecture
 import Component
 import Introspect
+import SettingFeature
 import SwiftUI
 import Styleguide
 
 public struct FavoritesScreen: View {
     private let store: Store<FavoritesState, FavoritesAction>
+    @ObservedObject private var viewStore: ViewStore<FavoritesState, FavoritesAction>
 
     public init(
         store: Store<FavoritesState, FavoritesAction>
     ) {
         self.store = store
+        self.viewStore = .init(store)
     }
 
     public var body: some View {
         NavigationView {
             ScrollView {
-                WithViewStore(store) { viewStore in
-                    LazyVGrid(
-                        columns: Array(
-                            repeating: GridItem(.flexible(), spacing: .zero),
-                            count: 2
-                        ),
-                        spacing: 16
-                    ) {
-                        ForEach(viewStore.items) { item in
-                            createCard(
-                                item: item,
-                                tapAction: {
-                                    viewStore.send(.tap(item))
-                                },
-                                tapFavoritesAction: {
-                                    viewStore.send(.favorite(item))
-                                }
-                            )
-                            .padding(8)
-                        }
-                    }
-                    .onAppear {
-                        viewStore.send(.refresh)
+                LazyVGrid(
+                    columns: Array(
+                        repeating: GridItem(.flexible(), spacing: .zero),
+                        count: 2
+                    ),
+                    spacing: 16
+                ) {
+                    ForEach(viewStore.items) { item in
+                        createCard(
+                            item: item,
+                            tapAction: {
+                                viewStore.send(.tap(item))
+                            },
+                            tapFavoritesAction: {
+                                viewStore.send(.favorite(item))
+                            }
+                        )
+                        .padding(8)
                     }
                 }
+                .onAppear {
+                    viewStore.send(.refresh)
+                }
                 .padding(.horizontal, 8)
             }
             .navigationBarTitle(L10n.FavoriteScreen.title, displayMode: .large)
             .navigationBarItems(
-                trailing: AssetImage.iconSetting.image
-                    .renderingMode(.template)
-                    .foregroundColor(AssetColor.Base.primary.color)
+                trailing: Button(action: {
+                    viewStore.send(.toggleSettings(true))
+                }) {
```

Comment:
Closure warning is happening. Please fix.

Reply:
DONE